### ISSUE TITLE
Composition /context/start_time and /end_time set to Unix epoch

### ISSUE THREAD

**Issue by grantforrester-nes** [ehrbase/ehrbase#59]
Creating a composition with a context/start_time (and /end_time) results in the Unix epoch being stored and returned.

Seen in v0.9.0.

**Steps To Reproduce**
1. POST a new composition to `ehrbase/rest/openehr/v1/ehr/{ehr_id}/composition` with context including start_time e.g
```
"context": {
    "_type": "EVENT_CONTEXT",
    "start_time": {
      "_type": "DV_DATE_TIME",
      "value": "2019-12-02T11:56:35Z00000000"
    }
}
```
2. GET the composition from `ehrbase/rest/openehr/v1/ehr/{ehr_id}/composition/{composition_id}`
3. Check the returned composition context.

**Expected value**
Expect the returned start_time to be...
```
"context": {
    "_type": "EVENT_CONTEXT",
    "start_time": {
      "_type": "DV_DATE_TIME",
      "value": "2019-12-02T11:56:35Z00000000"
    }
}
```

**Actual value**
The Unix epoch is returned
```
"context": {
    "_type": "EVENT_CONTEXT",
    "start_time": {
      "_type": "DV_DATE_TIME",
      "value": "1970-01-01T00:00:00Z00000000"
    }
}
```

**Comment by grantforrester-nes** [comment]
I have attached PR #60 to fix this issue

**Comment by chevalleyc** [comment]
Merged in main codebase with #83 . Many thanks !

**Comment by grantforrester-nes** [comment]
Thanks 👍
